origin-growth/package-lock.json

```diff
@@ -71,6 +71,34 @@
 			"resolved": "https://registry.npmjs.org/@protobufjs/utf8/-/utf8-1.1.0.tgz",
 			"integrity": "sha1-p3c2C1s5oaLlEG+OhY8v0tBgxXA="
 		},
+		"@sendgrid/client": {
```

Comment:
We don't have package-lock in version control in a most of the modules. @tomlinton do we have any general guidelines whether to leave `package-lock.json` in the modules or leave it out? 

I know that docker environment ignores this file since it doesn't copy it into containers. Not sure what to do about local environments.

Reply:
Yes, package-lock.json is not used anymore. But I think that the idea is that when adding a new dependency, we want to commit it because otherwise the next person that runs lerna bootstrap on that scope will see package-lock.json has changed  and might get confused as to why it was not committed.

Reply:
Ah yes, this makes a lot of sense for local development. Another thing we could do is add all `package-locks.json` files to gitignore